sources/packages/backend/libs/sims-db/src/entities/queue_configuration.model.ts

```diff
@@ -0,0 +1,32 @@
+import { QueueNames } from "@sims/services/queue";
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { TableNames } from "../constant";
+import { EXCEPTION_NAME_MAX_LENGTH } from "./application-exception-requests.model";
+import { QueueConfigurationDetails } from "./queue-configuration.type";
+import { RecordDataModel } from "./record.model";
+
+@Entity({ name: TableNames.QueueConfigurations })
+export class QueueConfiguration extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  /**
+   * Queue name.
+   */
+  @Column({
+    name: "name",
+    nullable: false,
+    length: EXCEPTION_NAME_MAX_LENGTH,
+  })
+  queueName: QueueNames;
+
+  /**
+   * Queue configuration.
+   */
+  @Column({
+    name: "configuration",
+    nullable: false,
```

Comment:
What if a queue must not have a retry? Configurations could be just null in that case.

Reply:
I was mentioning about the column being nullable when required. Is the idea is to change the column type to nullable when we see a queue in future that must not be retried?